dags/mantaray/run_mantaray_jobs.py

```diff
@@ -76,7 +79,7 @@ def run_test_code_on_persistent_TPUVM():
               sudo docker ps -a --filter \"name=testooo\" -q | grep -q . && sudo docker rm -f testooo; \
               sudo docker run --privileged --net host --shm-size=16G --name testooo \
               docker.io/vllm/vllm-tpu:270a5da495d24e947a71e2fa0c56635f4fad2dc3 bash -c \" \
-                  export HF_TOKEN=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 && \
```

Comment:
This token is already leaked to the internet. As a result, you'll need to go to https://huggingface.co/settings/tokens and delete this token. Otherwise any hacker can pretend to be you in the future.

Reply:
Thanks, deleted this token